test/e2e/tests/request-queuing/multiple-networks-dapps-txs.spec.js

```diff
@@ -23,8 +25,8 @@ describe('Request Queuing for Multiple Dapps and Txs on different networks.', fu
         dapp: true,
         fixtures: new FixtureBuilder()
           .withNetworkControllerDoubleGanache()
-          .withPermissionControllerConnectedToTwoTestDapps()
-          .withSelectedNetworkControllerPerDomain()
+          // .withPermissionControllerConnectedToTwoTestDapps()
+          // .withSelectedNetworkControllerPerDomain() // TODO: Add back when refreshing state doesn't revert per dapp selected network. Remove having to connect to dapp
```

Comment:
So what is this test actually testing now?

Reply:
by 
> refreshing state doesn't revert per dapp selected network.

I assume you mean the migration removing the `domains` state?

Reply:
What do you mean? The test still has multiple dapps on different networks and sending txs while on the different networks leads to switch network confirmations.

Reply:
hmmm okay I didn't actually read what those methods do only inferred by their names, but I'm confused how it works without `withSelectedNetworkControllerPerDomain`?

Reply:
yes that's what that method does so I'm confused why the network switching would happen any more? I wouldn't expect it to with `perDomainNetwork` === false

Reply:
Correct, and assigns the dapps to domains. But with the migration and state deletion, we just manually connect further down.